hdk/common/scripts/wait_for_afi.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2.7
+#!/usr/bin/env python
```

Comment:
This breaks python2.7. Add the following line before "import argparse":

`from __future__ import print_function`